.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: Continuous Integration
+
+on:
+  push:
+
+env:
+  AWS_REGION: ${{ vars.AWS_REGION }}
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  APP_ID: ${{ secrets.APP_ID }}
+  ENV_ID: ${{ secrets.ENV_ID }}
+  CONFIG_ID: ${{ secrets.CONFIG_ID }}
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Run tests
+        run: docker-compose -f "action-services.yml" run app bash -c "./scripts/test.sh"
+
+  deploy:
+    name: Deploy
+    if: github.ref_name == 'main'
```

Comment:
should you be checking`(github.ref_name == 'main' || github.head_ref == 'main')` or is that unnecessary?

Reply:
This should be fine, thanks for checking!